api/views/privaterelay.py

```diff
@@ -148,6 +258,17 @@ def runtime_data(request):
 
 
 @extend_schema(
+    tags=["privaterelay"],
+    parameters=[
+        OpenApiParameter(
+            name="Authorization",
+            required=True,
+            location="header",
+            examples=[OpenApiExample("bearer", "Bearer XXXX-ZZZZ")],
+            description="FXA Bearer Token. Can not be set in browsable API.",
```

Comment:
praise: thanks for adding this to make it clear to api doc readers they can't use this endpoint in the docs.